sdk-extensions/aws/src/test/java/io/opentelemetry/sdk/extension/aws/resource/SimpleHttpClientTest.java

```diff
@@ -49,28 +49,11 @@ void testFetchString() {
   void testFailedFetchString() {
     Map<String, String> requestPropertyMap = ImmutableMap.of("key1", "value1", "key2", "value2");
     String urlStr = String.format("http://localhost:%s%s", server.httpPort(), "/path");
-    JdkHttpClient jdkHttpClient = new JdkHttpClient();
-    String result = jdkHttpClient.fetchString("GET", urlStr, requestPropertyMap, null);
+    SimpleHttpClient httpClient = new SimpleHttpClient();
+    String result = httpClient.fetchString("GET", urlStr, requestPropertyMap, null);
     assertThat(result).isEmpty();
   }
 
-  @Test
-  void requestBody() {
```

Comment:
I deleted the code path taking a request body as that was only used by the moved-out xray sampler code